src/Lean/Server/References.lean

```diff
@@ -130,13 +131,18 @@ def identOf : Info → Option (RefIdent × Bool)
   | Info.ofFieldInfo fi => some (RefIdent.const fi.projName, false)
   | _ => none
 
+open Completion in
```

Comment:
Having to pull in `Completion` here felt a bit hackish to me, is there a good candidate parent file that I can migrate the `isBlacklisted` related functions to?

Reply:
When in doubt, use `Lean.Server.Utils` :)